transform/voxel_around_line.go

```diff
@@ -42,6 +42,10 @@ func GetSpatialIdsWithinRadiusOfLine(startPoint *object.Point, endPoint *object.
 		return nil, error
 	}
 
+	if radius == 0 {
```

Comment:
This does not apply to a minute radius like 0.01.
The cause is that hLayers and vLayers cannot be 0.
Please resolve this issue with removing this cause, without removing symptoms.